### ISSUE TITLE
Reject: Automated Segmentation of Complicated Cystic Renal Masses Using 3D V-Net Convolu

### ISSUE THREAD

**Issue by Mantrainment** [Mantrainment/neuroscience_newsletter#24]
Automated Segmentation of Complicated Cystic Renal Masses Using 3D V-Net Convolutional Neural Network on MRI.
---
To develop and test a convolutional neural network model for automated segmentation of complicated cystic renal masses (cCRMs) on MRI. This multicenter retrospective study analyzed 210 cCRMs between October 2019 and May 2021, divided into training/internal validation (n = 150, Institution 1) and test sets (n = 60, Institutions 2-4). Comparative 3D V-Net and U-Net models were developed across seven MRI sequences (T2-weighted, diffusion-weighted, apparent diffusion coefficient maps, unenhanced T1-weighted, and enhanced corticomedullary, nephrographic, and excretory phases images). A total of 14 models were developed, and seven pairwise comparisons were performed between the 3D V-Net and U-Net models. Segmentation performance was evaluated using Dice similarity coefficient (DSC) and Hausdorff distance (HD), with subgroup analysis of small cCRMs (≤40mm). In the test set, the excretory-phase V-Net (EPV-Net model) showed the highest DSC, and perform better than the corresponding U-Net (EPU-Net model) across all cCRMs (DSC: 0.74 ± 0.05 vs 0.70 ± 0.06, P < 0.001; HD: 27.41 ± 7.44 mm vs 39.18 ± 11.07 mm, P < 0.001) and the 35 small cCRMs subgroup (DSC: 0.74 ± 0.05 vs 0.70 ± 0.06, P < 0.001; HD: 27.48 mm ± 6.32 vs 38.72 ± 10.69 mm, P < 0.001). The 3D EPV-Net model demonstrated good segmentation accuracy, even for small lesions, supporting its clinical utility for cCRMs evaluation. This automated approach may streamline workflow compared to manual segmentation in cCRMs assessment.
